connection.go

```diff
@@ -269,6 +272,14 @@ type Opts struct {
 	Transport string
 	// SslOpts is used only if the Transport == 'ssl' is set.
 	Ssl SslOpts
+	// Minimal protocol version that should be supported by
+	// Go connection client and Tarantool server. By default
+	// it is equal to 0 (no restrictions)
+	RequiredProtocolVersion ProtocolVersion
+	// List of protocol features that should be supported by
+	// Go connection client and Tarantool server. By default
+	// it is a nil array (no restrictions)
+	RequiredProtocolFeatures []ProtocolFeature
```

Comment:
Could we replace it with?

RequiredServerInfo PrtocolInfo

Reply:
Yeah, it easily can be

Reply:
Replaced